### PR TITLE
Remove unused imports

### PR DESCRIPTION
**Pull request by michielderoos**
**Describe the Pull Request**

Been playing with [vulture](https://pypi.org/project/vulture/), and thought I should do a super quick PR to get rid of some unused imports. 

**Todo**

- [x] Link relevant [trello card](https://trello.com/b/PA2LhlOh/sempo-dev) or [related issue](https://github.com/teamsempo/SempoBlockchain/issues) to the pull request
- [x] Request review from relevant @person or team
- [x] QA your pull request. This includes running the app, login, testing changes etc.
- [x] Bump [backend](https://github.com/teamsempo/SempoBlockchain/blob/e3eb0f480e86d5a8ef2c1814127b70ff018671c4/config.py#L7) and/or [frontend](https://github.com/teamsempo/SempoBlockchain/blob/e3eb0f480e86d5a8ef2c1814127b70ff018671c4/app/package.json#L3) versions following Semver

### CODE REVIEW

**Comment by michielderoos** [comment]
[Remove unused imports](https://trello.com/c/rX6m9v9g/712-remove-unused-imports)

File: app/server/api/transfer_usage_api.py [hunk 1]
```diff
@@ -1,7 +1,6 @@
 from flask import Blueprint, request, make_response, jsonify
 from flask.views import MethodView
 from sqlalchemy import desc
```
Comment (tristanhcole) [inline app/server/api/transfer_usage_api.py:3]: `from sqlalchemy import desc` is also unused it seems?

File: app/server/utils/amazon_s3.py [hunk 2]
```diff
@@ -1,12 +1,10 @@
 import os
-import shutil
 import requests
 import datetime
 from flask import g, current_app
 import boto3
```
Comment (tristanhcole) [inline app/server/utils/amazon_s3.py:5]: `import boto3` also unused

File: app/server/utils/metrics/filters.py [hunk 3]
```diff
@@ -147,27 +147,9 @@ def _apply_ca_filters(query, filters, user_join_condition):
...
-    CreditTransfer.transfer_type == TransferTypeEnum.EXCHANGE,
-]
-
-beneficiary_filters = [User.has_beneficiary_role == True]
-vendor_filters = [User.has_vendor_role == True]
-
-exhaused_balance_filters = [
-    CreditTransfer.transfer_type == TransferTypeEnum.PAYMENT,
-    TransferAccount._balance_wei == 0
-]
```
Comment (tristanhcole) [inline app/server/utils/metrics/filters.py:172]: would these be used in future?

**Review by tristanhcole — APPROVED** [review]
went through manually - looks reasonable

a few comments, but otherwise lgtm

File: app/server/api/transfer_usage_api.py (hunk 1 above)
Comment (michielderoos) [reply app/server/api/transfer_usage_api.py:3]: Hm, vulture didn't pick those up for some reason. Will delete when I log on in the AM!

File: app/server/utils/metrics/filters.py (hunk 3 above)
Comment (michielderoos) [reply app/server/utils/metrics/filters.py:172]: I don't imagine they will be.They're pretty self explanatory to rewrite if we do though IMO

Comment (tristanhcole) [reply app/server/utils/metrics/filters.py:172]: ok!

File: app/server/api/transfer_usage_api.py (hunk 1 above)
Comment (tristanhcole) [reply app/server/api/transfer_usage_api.py:3]: no worries!

**Pull request merged**